saleor/checkout/calculations.py

```diff
@@ -517,6 +525,15 @@ def _apply_tax_data(
     if not tax_data:
         return
 
+    if check_negative_values_in_tax_data(tax_data):
+        logger.error(
```

Comment:
I am not sure if it should be `error` or `warning` level

Reply:
I think that error is ok, as it's an anomaly that shouldn;t happened